### ISSUE TITLE
[ActiveJob] Cannot use `queue_as` for select queue, only sidekiq options are supported (first run)

### ISSUE THREAD

**Issue by arathunku** [sidekiq-cron/sidekiq-cron#437]
Hi :wave: 

looks like sidekiq-cron works "okayish" with ActiveJob. It detects and runs jobs:)

There are 2 problems I'm running into

1. [here](https://github.com/sidekiq-cron/sidekiq-cron/blob/78eb493adbfeb908861ec9b3a76e5e7a01adf339/lib/sidekiq/cron/job.rb#L322) only sidekiq options are fetched on init, if sidekiq options change later on, this will not be fetched
2. When some of the options change, for example queue, it will be **only** overwritten if passed by args [see here](https://github.com/sidekiq-cron/sidekiq-cron/blob/78eb493adbfeb908861ec9b3a76e5e7a01adf339/lib/sidekiq/cron/job.rb#L334).

No code here supports ActiveJob configuration.

Is this more of a bug and PRs to get better ActiveJob is welcome or there's something wrong I'm doing?

Repro is very minimal

```yml
  foobar:
    cron: "*/5 * * * *" 
    class: "FooBar"
```

```rb
  class FooBar < ApplicationJob
    queue_as :low

    def perform
```

**Comment by markets** [comment]
Hey @arathunku 👋

In #424 we are discussing some changes to properly use AJ. Do you think those changes can help with this issue too?

**Comment by arathunku** [comment]
I've seen the issue but decided to open a new issue because I found no need for any global options nor `active_job: true`. Detection of ActiveJob jobs by `is_active_job?` works for enquing jobs and so far, I've only found the two missing parts documented above.

I prefer to see options on "job" level as much as possible, where you'd usually make a change and follow the same patterns both for non-cron-job Jobs and normal background jobs. 

For CronJobs, we also set up additional defaults via:

```ruby
  class SetupCronJobDefaults
    def self.setup_for_all
      ConfigLoader.load("sidekiq-cron.yml").each do |(_name, job)|
        klass = Sidekiq::Cron::Support.constantize(job.fetch("class", job["klass"]))
         .....
      end
    end
  end
```
This is executed before config is loaded.

So that nobody needs to remember about some incantation for every job definition.
